Recast/HomeViewController.swift

```diff
@@ -0,0 +1,274 @@
+            image: #imageLiteral(resourceName: "series_img_1"),
+            title: "110: Asking All the Questions",
+            date: "September 13, 2018",
+            duration: 58,
+            timeLeft: 10,
+            isNew: true)
+
+    let dummyContinue2 = DummyPodcastSeries(
+        image: #imageLiteral(resourceName: "series_img_2"),
+        title: "Dummy Podcast 2",
+        date: "November 1st, 2018",
+        duration: 10,
+        timeLeft: 29)
+
+    var homeTableView: UITableView!
+
+    /// Maps section/row index to the appropriate header type
+    let headerTypes: [Int: HomeSectionType] = [
+        0: .continueListening,
+        1: .yourFavorites,
+        2: .browseTopics
+    ]
+
+    /// Maps tag of each collection view (nested inside each table view cell) to the corresponding HomeSectionType
+    let collectionViewTypes: [Int: HomeSectionType] = [
+        100: .continueListening,
+        101: .yourFavorites,
+        102: .browseTopics
+    ]
+
+    /// Array of podcasts for the "continue listening" section
+    var continuePodcasts: [DummyPodcastSeries] = [] {
+        didSet {
+            guard let cv = view.viewWithTag(100) as? UICollectionView else { return }
+            cv.reloadData()
+        }
+    }
+
+    /// Array of podcasts for the "your favorites" section
+    var favoritePodcasts: [DummyPodcastSeries] = [] {
+        didSet {
+            guard let cv = view.viewWithTag(101) as? UICollectionView else { return }
+            cv.reloadData()
+        }
+    }
+
+    /// Array of podcasts for the "recommended for you" section
+    var recommendedPodcasts: [DummyPodcastSeries] = [] {
+        didSet {
+            guard let cv = view.viewWithTag(102) as? UICollectionView else { return }
+            cv.reloadData()
+        }
+    }
+
+    // Table View reuse identifiers
+    let continueTvReuse = "continueListeningTvReuse"
+    let gridTvReuse = "gridTvReuse"
+
+    // Collection View reuse identifiers
+    let continueCvReuse = "continueListeningCvReuse"
+    let gridCvReuse = "gridCvReuse"
+
+    // MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // VC settings
+        title = "Home"
+        view.backgroundColor = .black
+        navigationController?.navigationBar.prefersLargeTitles = true
+
+        // homeTableView
+        homeTableView = UITableView(frame: .zero, style: .grouped)
+        homeTableView.delegate = self
+        homeTableView.dataSource = self
+        homeTableView.allowsSelection = false
+        homeTableView.backgroundColor = .black
+        homeTableView.separatorColor = .black
+        homeTableView.register(ContinueTableViewCell.self, forCellReuseIdentifier: continueTvReuse)
+        homeTableView.register(PodcastGridTableViewCell.self, forCellReuseIdentifier: gridTvReuse)
+        view.addSubview(homeTableView)
+
+        prepareDummy()
+        setUpConstraints()
+    }
+
+    func setUpConstraints() {
```

Comment:
i named my func `makeConstraints()`, which do we think is better @drewsdunne @mindylou ?

Reply:
I don't have a preference but it should be the same across all views

Reply:
lets do `setUpConstraints` so we dont confuse with the snapkit func